lib/auth/v2/headerAuthCheck.js

```diff
@@ -15,7 +15,7 @@ function check(request, log, data) {
     timestamp = Date.parse(timestamp);
     if (!timestamp) {
         log.debug('missing security header: invalid date/timestamp');
-        return { err: errors.MissingSecurityHeader };
+        return { err: errors.AccessDenied };
```

Comment:
I don't think there is a loss of performance by just creating a new Error instance, but the request has already errored out at this point, so we don't have to worry about it. And we can't re-use the available instance to customize the error as it will update the error description in global scope.
Even if this doesn't happen often it's going to help that one time `AccessDenied` is sent to the client and the error is too vague to interpret what actually happened.